lib/ansible/executor/action_write_locks.py

```diff
@@ -36,8 +37,7 @@
 
     # These plugins are called directly by action plugins (not going through
     # a strategy).  We precreate them here as an optimization
-    mods = set(p['name'] for p in Facts.PKG_MGRS)
+    mods = set(C.PKG_MGR_MODULES)
```

Comment:
@abadger in this branch, only action_write_locks uses PKG_MGR_MODULES and the only places I see action_write_locks is in strategy/__init__.py and module_common._find_module_utils which both only seem to be used controller side. 

And the only place PKG_MGRS is used is in lib/ansible/module_utils/facts/system/pkg_mgr.py which is only used remote side.